src/start.js

```diff
@@ -11,3 +11,11 @@ export function start() {
 export function isStarted() {
 	return started;
 }
+
+const startWarningDelay = 3000;
```

Comment:
👍  I'll change it to 5 seconds. As for the ability to suppress the warning, let's add that in once someone actually runs into that situation.